docs/projects/adrv9009/index.rst

```diff
@@ -0,0 +1,445 @@
+   +-------------------+------------------------------------------------------+
+   | TX_JESD_M         |                          4                           |
+   +-------------------+------------------------------------------------------+
+   | TX_JESD_L         |                          4                           |
+   +-------------------+------------------------------------------------------+
+   | TX_JESD_S         |                          1                           |
+   +-------------------+------------------------------------------------------+
+   | RX_OS_JESD_M      |                          2                           |
+   +-------------------+------------------------------------------------------+
+   | RX_OS_JESD_L      |                          2                           |
+   +-------------------+------------------------------------------------------+
+   | RX_OS_JESD_S      |                          1                           |
+   +-------------------+------------------------------------------------------+
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Other considerations
+-------------------------------------------------------------------------------
+
+ADC - lane mapping
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Due to physical constraints, Rx lanes are reordered as described in the
+following table.
+
+=============== ===========================
+ADC phy Lane    FPGA Rx lane / Logical Lane
+=============== ===========================
+0               0
+1               1
+=============== ===========================
+
+================ ===========================
+ADC OBS phy Lane FPGA Rx lane / Logical Lane
+================ ===========================
+0                2
+1                3
+================ ===========================
+
+DAC - lane mapping
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Due to physical constraints, Tx lanes are reordered as described in the
+following table.
+
+============ ===========================
+DAC phy lane FPGA Tx lane / Logical lane
+============ ===========================
+0            3
+1            2
+2            0
+3            1
+============ ===========================
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :dokuwiki:`[Wiki] ADRV9009 & ADRV9008 Prototyping Platform User Guide <resources/eval/user-guides/adrv9009>`
+
+Here you can find the quick start guides available for these evaluation boards:
+
+.. list-table::
+   :widths: 20 10
+   :header-rows: 1
+
+   * - Evaluation board
+     - Zynq UltraScale+ MP
+   * - ADRV9009/ADRV9008
+     - :dokuwiki:`ZCU102 <resources/eval/user-guides/adrv9009/quickstart/zynqmp>`
```

Comment:
Should reference the system level documentation, since we already have an user guide there